specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -40,6 +41,13 @@ import { InvalidMergeRecordsDialog } from './InvalidMergeRecords';
 import { mergingQueryParameter } from './queryString';
 import { Status } from './Status';
 
+export const recordMergingTables = new Set<keyof Tables>([
```

Comment:
query builder should also use this list of tables (don't know what list it does right now)

Reply:
It uses this https://github.com/specify/specify7/blob/9044ffc1e29eea13e1d18ccf20cad10ef5290978/specifyweb/frontend/js_src/lib/components/Merging/definitions.ts#L5-L21. 
@CarolineDenis can you make them the same? The reason was because to prevent group members merge (and it is generic enough to handle cases for other tables)